docs/admin/master-node-communication.md

```diff
@@ -83,3 +83,57 @@ cluster (connecting to the ssh server listening on port 22) and passes all
 traffic destined for a kubelet, node, pod, or service through the tunnel.
 This tunnel ensures that the traffic is not exposed outside of the private
 GCE network in which the cluster is running.
+
+### Kubelet TLS Bootstrap 
+
+The 1.4 release introduces an experimental API for requesting certificates from a cluster-level CA. The first supported use of this API is the provisioning of TLS client certificates for kubelets. The proposal can be found [here] and progress on the feature is being tracked as [feature #43](https://github.com/kubernetes/features/issues/43). 
```

Comment:
Couple of style things for clarity:
- Please refer to the API by name and avoid pronoun usage. E.g. "Kubernetes 1.4 introduces the experimental <API name> API for requesting certificates..." and "The first supported use of the <API name> API is to provision TLS client certificates for kubelets."
- Please spell out the "CA" acronym at first use.